neurom/features/neuritefunc.py

```diff
@@ -179,57 +178,6 @@ def pl2(node):
     return _map_sections(pl2, neurites, neurite_type=neurite_type)
 
 
-################################################################################
-# Features returning one value per NEURON                                      #
-################################################################################
-
-def _map_neurons(fun, neurites, neurite_type):
```

Comment:
@mgeplf FYI. I saw that you added this code in 2016, so just for safety I'm notifying you.

Reply:
Thanks for the heads up.

Before than they were just `partial` wrapped functions; this way there could be documentation added to them that allowed for better docs/cli info, iirc. As long as the behavior is the same or better, I have no problems w/ the refactorings.